app/src/main/java/kr/co/nottodo/presentation/toplevel/recommendation/data/recommendationlistdata/RecommendationParentData.kt

```diff
@@ -0,0 +1,7 @@
+package kr.co.nottodo.presentation.toplevel.recommendation.data.recommendationlistdata
+
+data class RecommendationParentData(
+    val id: Int,
+    val title: String,
+    val childDatas: List<RecommendationChildData>
+)
```

Comment:
```suggestion
data class RecommendationParentData(
    val id: Int,
    val title: String,
    val childDatas: List<RecommendationChildData>
) {
    data class RecommendationChildData(
        val id: Int,
        val title: String,
   )
}
```

이런식으로 중괄호 안에 클래스로 선언해서 사용할 수 있음(이렇게 사용하면 위에서 사용한 RecommendationChildData는 지워도 됨)